projects/admin/src/app/routes/statistics-cfg-route.ts

```diff
@@ -15,65 +15,166 @@
-        { path: 'edit/:pid', component: EditorComponent, canActivate: [ CanAccessGuard ], data: { action: CAN_ACCESS_ACTIONS.UPDATE } },
-        { path: 'new', component: EditorComponent, canActivate: [ PermissionGuard ], data: { permissions: [ PERMISSIONS.STAT_CFG_CREATE ] } }
+        {
+          path: "",
+          component: RecordSearchPageComponent,
+          canActivate: [PermissionGuard],
+          data: {
+            permissions: [
+              PERMISSIONS.STAT_CFG_ACCESS,
+              PERMISSIONS.STAT_CFG_SEARCH,
+            ],
+            operator: PERMISSION_OPERATOR.AND,
+          },
+        },
+        {
+          path: "detail/:pid",
+          component: DetailComponent,
+          canActivate: [CanAccessGuard],
+          data: { action: CAN_ACCESS_ACTIONS.READ },
+        },
+        {
+          path: "edit/:pid",
+          component: EditorComponent,
+          canActivate: [CanAccessGuard],
+          data: { action: CAN_ACCESS_ACTIONS.UPDATE },
+        },
+        {
+          path: "new",
+          component: EditorComponent,
+          canActivate: [PermissionGuard],
+          data: { permissions: [PERMISSIONS.STAT_CFG_CREATE] },
+        },
       ],
       data: {
         types: [
           {
             key: this.name,
-            label: _('Statistics configuration'),
+            label: _("Statistics configuration"),
             editorSettings: {
-              longMode: true,
+              longMode: false,
             },
             component: StatisticsCfgBriefViewComponent,
             detailComponent: StatisticsCfgDetailViewComponent,
             aggregationsBucketSize: 10,
-            aggregationsExpand: [
-              'category'
-            ],
-            aggregationsOrder: [
-              'category'
-            ],
-            canAdd: () => of({ can: this._routeToolService.permissionsService.canAccess(PERMISSIONS.STAT_CFG_CREATE) }),
-            permissions: (record: any) => this._routeToolService.permissions(record, this.recordType),
+            aggregationsExpand: ["category"],
+            aggregationsOrder: ["category"],
+            canAdd: () =>
+              of({
+                can: this._routeToolService.permissionsService.canAccess(
+                  PERMISSIONS.STAT_CFG_CREATE
+                ),
+              }),
+            permissions: (record: any) =>
+              this._routeToolService.permissions(record, this.recordType),
+            formFieldMap: (
+              field: FormlyFieldConfig,
+              jsonSchema: JSONSchema7
+            ): FormlyFieldConfig => {
+              const formWidget = jsonSchema.widget;
+              if (
+                formWidget?.formlyConfig?.templateOptions?.fieldMap ===
+                "libraries"
+              ) {
+                field.type = "select";
+                field.hooks = {
+                  ...field.hooks,
```

Comment:
For readability, maybe i will extract this into a dedicated function. Like `_populateLocationsByCurrentUserLibrary()` function into `items_routes.ts`